src/Tests/ObjectLifetimeTests/ObjectLifetimeTests.Lifted/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+    <Import Project="ObjectLifetimeTests.Lifted\CSWinRTInApp.props"   />
+    <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+    <Import Project="..\..\..\..\nuget\Microsoft.Windows.CsWinRT.props" />
```

Comment:
Any reason why the one in the root directory props doesn't automatically apply here?  Is it because we don't import directory.build.props at a certain folder level and if so why?

Reply:
The reason was because the `SimulateCsWinRTNugetReference` property was set after the import statement. After adding the property group before the import statement there no longer needs to be a direct reference to the `Microsoft.Windows.CsWinRT.props` file.